scripts/install_chrome_for_ci.py

```diff
@@ -80,8 +79,7 @@ def main():
                 'Version {} was found instead.'
             ).format(CHROME_VERSION, found_version)
         )
-    python_utils.PRINT(
-        'Chrome version {} installed.'.format(found_version))
+    print('Refer to https://github.com/oppia/oppia/wiki/Troubleshooting')
```

Comment:
Sorry, wrong suggestion please set it back to the original.

Reply:
done